src/transforms/operationTransforms.ts

```diff
@@ -257,6 +257,9 @@ export async function transformOperation(
     usedModels: [typeName]
   };
 
+  const isLRO: boolean =
+    operation.extensions && operation.extensions["x-ms-long-running-operation"];
```

Comment:
Does this need to be wrapped in a Boolean?
```suggestion
    Boolean(operation.extensions && operation.extensions["x-ms-long-running-operation"]);
```
Otherwise if either item were undefined, wouldn't `isLRO` be undefined too?

Reply:
updated, thanks!